docs/content/system-update-order.md

```diff
@@ -0,0 +1,108 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+
+## System update order in the Unity GDK
+
+Unity provides attributes to define the  update order of systems. These attributes are: `UpdateInGroup`, `UpdateBefore` and `UpdateAfter`. Note: You can only have one attribute of each type assigned to a system. If multiple are assigned they might override each other.
```

Comment:
- remove double space between "the" and "update"
- add the link to the Unity docs: https://github.com/Unity-Technologies/EntityComponentSystemSamples/blob/master/Documentation/content/ecs_in_detail.md#system-update-order (link the words "update order of systems")

Reply:
Also, for the "Note", it might be best to have this on a new line and use `>` to indent it.

Reply:
👍 Changed